code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -216,11 +216,19 @@
 		else
 			msg += "<B>[t_He] [t_has] severe cellular damage.</B>\n"
 
-
+	var/list/missing = list("head", "chest", "l_arm", "r_arm", "l_leg", "r_leg")
```

Comment:
this l_arm and r_arm etc bullshit isn't needed ree you said it aswell, change it to right leg etc